src/qd8-f32-qb4w-gemm/gen/qd8-f32-qb4w-gemm-6x16-minmax-neon-mlal-lane.c

```diff
@@ -0,0 +1,930 @@
+      const int16x8_t vxa4 = vmovl_s8(va4);
+      const int8x8_t va5 = vld1_s8(a5); a5 += 8;
+      const int16x8_t vxa5 = vmovl_s8(va5);
+
+      const int8x8_t vb01234567c01 = vld1_s8(w); w = (const uint8_t*) w + 8;
+      const int8x8_t vb89ABCDEFc01 = vld1_s8(w); w = (const uint8_t*) w + 8;
+      const int8x8_t vb01234567c23 = vld1_s8(w); w = (const uint8_t*) w + 8;
+      const int8x8_t vb89ABCDEFc23 = vld1_s8(w); w = (const uint8_t*) w + 8;
+      const int8x8_t vb01234567c45 = vld1_s8(w); w = (const uint8_t*) w + 8;
+      const int8x8_t vb89ABCDEFc45 = vld1_s8(w); w = (const uint8_t*) w + 8;
+      const int8x8_t vb01234567c67 = vld1_s8(w); w = (const uint8_t*) w + 8;
+      const int8x8_t vb89ABCDEFc67 = vld1_s8(w); w = (const uint8_t*) w + 8;
+      const int8x8_t vb01234567c0 = vshl_n_s8(vb01234567c01, 4);
+      const int8x8_t vb01234567c1 = vand_s8(vb01234567c01, vmask);
+      const int8x8_t vb89ABCDEFc0 = vshl_n_s8(vb89ABCDEFc01, 4);
+      const int8x8_t vb89ABCDEFc1 = vand_s8(vb89ABCDEFc01, vmask);
+      const int8x8_t vb01234567c2 = vshl_n_s8(vb01234567c23, 4);
+      const int8x8_t vb01234567c3 = vand_s8(vb01234567c23, vmask);
+      const int8x8_t vb89ABCDEFc2 = vshl_n_s8(vb89ABCDEFc23, 4);
+      const int8x8_t vb89ABCDEFc3 = vand_s8(vb89ABCDEFc23, vmask);
+      const int8x8_t vb01234567c4 = vshl_n_s8(vb01234567c45, 4);
+      const int8x8_t vb01234567c5 = vand_s8(vb01234567c45, vmask);
+      const int8x8_t vb89ABCDEFc4 = vshl_n_s8(vb89ABCDEFc45, 4);
+      const int8x8_t vb89ABCDEFc5 = vand_s8(vb89ABCDEFc45, vmask);
+      const int8x8_t vb01234567c6 = vshl_n_s8(vb01234567c67, 4);
+      const int8x8_t vb01234567c7 = vand_s8(vb01234567c67, vmask);
+      const int8x8_t vb89ABCDEFc6 = vshl_n_s8(vb89ABCDEFc67, 4);
+      const int8x8_t vb89ABCDEFc7 = vand_s8(vb89ABCDEFc67, vmask);
+
+      const int16x8_t vxb01234567c0 = vmovl_s8(vb01234567c0);
+      const int16x8_t vxb89ABCDEFc0 = vmovl_s8(vb89ABCDEFc0);
+      const int16x8_t vxb01234567c1 = vmovl_s8(vb01234567c1);
+      const int16x8_t vxb89ABCDEFc1 = vmovl_s8(vb89ABCDEFc1);
+      const int16x8_t vxb01234567c2 = vmovl_s8(vb01234567c2);
+      const int16x8_t vxb89ABCDEFc2 = vmovl_s8(vb89ABCDEFc2);
+      const int16x8_t vxb01234567c3 = vmovl_s8(vb01234567c3);
+      const int16x8_t vxb89ABCDEFc3 = vmovl_s8(vb89ABCDEFc3);
+      const int16x8_t vxb01234567c4 = vmovl_s8(vb01234567c4);
+      const int16x8_t vxb89ABCDEFc4 = vmovl_s8(vb89ABCDEFc4);
+      const int16x8_t vxb01234567c5 = vmovl_s8(vb01234567c5);
+      const int16x8_t vxb89ABCDEFc5 = vmovl_s8(vb89ABCDEFc5);
+      const int16x8_t vxb01234567c6 = vmovl_s8(vb01234567c6);
+      const int16x8_t vxb89ABCDEFc6 = vmovl_s8(vb89ABCDEFc6);
+      const int16x8_t vxb01234567c7 = vmovl_s8(vb01234567c7);
+      const int16x8_t vxb89ABCDEFc7 = vmovl_s8(vb89ABCDEFc7);
+
+
+      vacc0x0123 = vmlal_lane_s16(vacc0x0123, vget_low_s16(vxb01234567c0), vget_low_s16(vxa0), 0);
```

Comment:
tell me how much spill with this large tile size IIRC this is what they have in the gemm-config?